akka-http-tests/src/test/java/docs/http/javadsl/server/HandlerExampleDocTest.java

```diff
@@ -248,43 +146,56 @@ public void testDeferredResultAsyncHandler() {
         }
         //#async-service-definition
 
+        // FIXME rewrite this beyond just fixing compiler errors, so it actually looks nice.
+        // e.g. extract getting the x and y param into a reusable method, like requestvals did before.
```

Comment:
This could be a TODO to inspire documentation improvements, which should now probably also synchronized between Scala and Java.